common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -401,3 +404,49 @@ def test_only_expected_fields_are_displayed(self):
         expected_fields = self.advanced_settings.expected_settings_names
         displayed_fields = self.advanced_settings.displayed_settings_names
         self.assertEquals(set(displayed_fields), set(expected_fields))
+
+
+@attr('shard_1')
+class ContentLicenseTest(StudioCourseTest):
+    def setUp(self):
+        super(ContentLicenseTest, self).setUp()
+        self.outline_page = CourseOutlinePage(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+        self.settings_page = SettingsPage(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+        self.lms_courseware = CoursewarePage(
+            self.browser,
+            self.course_id,
+        )
+        self.outline_page.visit()
+
+    def test_empty_license(self):
+        self.assertEqual(self.outline_page.license, "None")
```

Comment:
We use the convention of having BDD specs as the doc strings for each Bok Choy test, so the reader can see the purpose of the test. 

See https://openedx.atlassian.net/wiki/display/TNL/Code+Review+Guidelines+and+Gotchas